src/main/kotlin/no/nav/syfo/aktivitetskrav/database/AktivitetskravRepository.kt

```diff
@@ -0,0 +1,94 @@
+package no.nav.syfo.aktivitetskrav.database
+
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.application.database.toList
+import no.nav.syfo.domain.PersonIdent
+import java.sql.Connection
+import java.sql.ResultSet
+import java.time.OffsetDateTime
+import java.util.*
+
+class AktivitetskravRepository(private val database: DatabaseInterface) {
+
+    fun getAktivitetskrav(uuid: UUID): PAktivitetskrav? =
+        database.connection.use { connection ->
+            connection.prepareStatement(getAktivitetskravByUuidQuery).use {
+                it.setString(1, uuid.toString())
+                it.executeQuery().toList { toPAktivitetskrav() }.firstOrNull()
+            }?.run {
+                val vurderinger = getAktivitetskravVurderinger(aktivitetskravId = id, connection)
+                copy(vurderinger = vurderinger)
+            }
+        }
+
+    fun getAktivitetskrav(
+        personIdent: PersonIdent,
+    ): List<PAktivitetskrav> =
+        database.connection.use { connection ->
+            connection.prepareStatement(getAktivitetskravByPersonidentQuery).use {
+                it.setString(1, personIdent.value)
+                it.executeQuery().toList { toPAktivitetskrav() }
+            }.map {
+                val vurderinger = getAktivitetskravVurderinger(aktivitetskravId = it.id, connection)
+                it.copy(vurderinger = vurderinger)
+            }
+        }
+
+    private fun getAktivitetskravVurderinger(
+        aktivitetskravId: Int,
+        connection: Connection,
+    ): List<PAktivitetskravVurdering> =
+        connection.prepareStatement(getAktivitetskravVurderingerQuery).use {
+            it.setInt(1, aktivitetskravId)
+            it.executeQuery().toList { toPAktivitetskravVurdering() }
+        }
+
+    companion object {
+
+        private const val getAktivitetskravByUuidQuery =
```

Comment:
Hvilke fordeler får vi ved at disse er inne i et companion object, når de uansett er private? Ser også at IntelliJ anbefaler at variabler som er `const val` burde være all upper-case.

Reply:
> Hvilke fordeler får vi ved at disse er inne i et companion object, når de uansett er private? Ser også at IntelliJ anbefaler at variabler som er `const val` burde være all upper-case.

Da syns jeg vi skal gå for best practice, endrer til uppercase snakecase